docs/ref/models/instances.txt

```diff
@@ -309,6 +309,17 @@ pass a dictionary mapping field names to errors::
         'pub_date': ValidationError(_('Invalid date.'), code='invalid'),
     })
 
+.. note::
+   If the model is used in a model form anywhere on your site, you
+   must only use field names in your dictionary that will exist on all
+   forms, otherwise Django will crash because it will be unable to
```

Comment:
Thanks for the feedback! I now mention that the exception is a value error, and give some additional advice when to use this.